ext/natives/codegen_cfx_natives.lua

```diff
@@ -1299,6 +1299,40 @@ native 'SET_VEHICLE_WHEEL_XROT'
 	apiset 'client'
 	returns 'void'
 
+native 'GET_VEHICLE_WHEEL_TRACK_WIDTH'
+	arguments {
+		Vehicle 'vehicle',
+		int 'wheelIndex'
+	}
+	apiset 'client'
+	returns 'float'
+
+native 'SET_VEHICLE_WHEEL_TRACK_WIDTH'
+	arguments {
+		Vehicle 'vehicle',
+		int 'wheelIndex',
+		float 'value'
+	}
+	apiset 'client'
+	returns 'void'
+
+native 'GET_VEHICLE_WHEEL_CAMBER'
+	arguments {
+		Vehicle 'vehicle',
+		int 'wheelIndex'
+	}
+	apiset 'client'
+	returns 'float'
+
+native 'SET_VEHICLE_WHEEL_CAMBER'
```

Comment:
If this name is more accurate than XROT, rename XROT (and, in case someone else used it, add a compatibility alias - presumably would need a `hash` override and `alias` field in codegen definitions? would need verification by someone familiar with codegen).